src/PrimeTower.cpp

```diff
@@ -26,6 +26,13 @@ PrimeTower::PrimeTower()
 : wipe_from_middle(false)
 {
     const Scene& scene = Application::getInstance().current_slice->scene;
+
+    {
+        EPlatformAdhesion adhesion_type = scene.current_mesh_group->settings.get<EPlatformAdhesion>("adhesion_type");
+
+        multiple_extruders_on_first_layer = scene.settings.get<bool>("machine_extruders_share_nozzle") && ((adhesion_type == EPlatformAdhesion::RAFT) || (adhesion_type == EPlatformAdhesion::NONE));
```

Comment:
This line is the reason why the tests are failing. You're requesting this setting from the scene, i.e. global settings. The LayerPlan test only sets the mesh group settings (settings for a group of meshes that get sliced together from bottom to top, i.e. the entire scene unless using one-at-a-time mode).

```suggestion
        multiple_extruders_on_first_layer = scene.current_mesh_group->settings.get<bool>("machine_extruders_share_nozzle") && ((adhesion_type == EPlatformAdhesion::RAFT) || (adhesion_type == EPlatformAdhesion::NONE));
```